server/src/main/java/org/apache/druid/client/indexing/ClientCompactQueryTuningConfig.java

```diff
@@ -31,15 +31,15 @@
 {
   // These default values should be synchronized with those of IndexTuningConfig
   private static final int DEFAULT_MAX_ROWS_IN_MEMORY = 75_000;
```

Comment:
This value got changed to 1_000_000, may should get this from [here](https://github.com/apache/incubator-druid/blob/master/server/src/main/java/org/apache/druid/segment/indexing/TuningConfig.java#L37). Unless you need it to be 75_000.

Reply:
Thanks. I found this is not necessary because they will be set to defaults in `IndexTuningConfig` if they are null. Removed all default values.